<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>basic_gameClassClass</class>
 <widget class="QWidget" name="basic_gameClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>欢乐连连看——基本模式</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/xiaowang/.designer/backup/LLK.ico</normaloff>C:/Users/xiaowang/.designer/backup/LLK.ico</iconset>
  </property>
  <widget class="QPushButton" name="pb_begin_game">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_stop_game">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_showinfor_game">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>180</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>提示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_refresh_game">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>250</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>重排</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setting">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>530</y>
     <width>45</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helper">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>570</y>
     <width>45</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>400</height>
    </size>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>640</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximum">
    <number>400</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
